### PR TITLE
[IMP] account_vat_period_end_statement: add multicompany rule

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: account_vat_period_end_statement/i18n/it.po [hunk 1]
```diff
@@ -213,13 +214,13 @@ msgid "Draft"
 msgstr "Bozza"
 
 #. module: account_vat_period_end_statement
-#: code:addons/account_vat_period_end_statement/models/account.py:448
+#: code:addons/account_vat_period_end_statement/models/account.py:451
 #, python-format
 msgid "Encoding error"
 msgstr "Encoding error"
```
Comment (OpenCode) [inline account_vat_period_end_statement/i18n/it.po]: Can you translate it?

File: account_vat_period_end_statement/migrations/8.0.3.0.0/post-migration.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+def migrate(cr, version):
+    if not version:
+        return
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        end_statements = env['account.vat.period.end.statement'].search([])
+        for end_statement in end_statements:
+            companies = end_statement.mapped('period_ids.company_id')
+            company_id = companies and companies[0] and companies[0].id or 1
```
Comment (OpenCode) [inline account_vat_period_end_statement/migrations/8.0.3.0.0/post-migration.py]: Search a company instead of use directly "1"

Comment (OpenCode) [inline account_vat_period_end_statement/migrations/8.0.3.0.0/post-migration.py]: You use directly "1". What happens if i delete the strandard company and create a new one?

File: account_vat_period_end_statement/i18n/it.po (hunk 1 above)
Comment (OpenCode) [inline account_vat_period_end_statement/i18n/it.po]: Ok

File: account_vat_period_end_statement/migrations/8.0.3.0.0/post-migration.py (hunk 2 above)
Comment (tafaRU) [inline account_vat_period_end_statement/migrations/8.0.3.0.0/post-migration.py]: I'm not sure 100%, if you prefer I can modify as written above.
Let me know!

Comment (OpenCode) [inline account_vat_period_end_statement/migrations/8.0.3.0.0/post-migration.py]: I prefer it. For security ;)

Comment (tafaRU) [inline account_vat_period_end_statement/migrations/8.0.3.0.0/post-migration.py]: @OpenCode, done, please see new diff.

**Comment by eLBati** [comment]
lint errors not related to these changes

**Pull request merged**